### ISSUE TITLE
Mask for recessive gene action

### ISSUE THREAD

**Issue by matteofigliuzzi** [rgcgithub/regenie#334]
Hello,

as far as I understand, the comphet mask applies a threshold of 2 to the mask from the sum rule. 

To identify recessive gene associations, it would be very useful a further mask which checks if the sum of alternate alleles within a gene is greater or equal than 2:

Single variant sites       recessive mask
00000000000000                0
00010000000000                0
00010000010000                1
00010000010010                1
00020000000000                1
00020000010000                1

In other words this "recessive mask" checks if the necessary (not sufficient) conditions for compound heterozygosity hold.
Is there any feature like that?

Thanks a lot,
Matteo

**Comment by joellembatchou** [comment]
Hi Matteo,

You can use `--build-mask comphet` along with `--test recessive` so once it builds the comphet mask it will apply recessive coding to it (so only individuals with 2 or more ALT alleles across sites will be coded as 1)

Cheers,
Joelle
